scripts/install/opt/seagate/cortx/motr/bin/motr_setup

```diff
@@ -123,7 +123,7 @@ class PostInstallCmd(Cmd):
             config_lvm(self)
         else:
             self.logger.error("There are LVM configurations present on the system. "
-                              "Please wipe them and re-run the step.\n")
```

Comment:
Can we ensure here, if they are valid LVM's or not.
If not then exit here with message as clear the LVM's and created the valid ones.

Reply:
We list LV names from lvdisplay and doing some grepping on it to extract LV path.
Once we get all LV paths, we check with cortx paths like '/dev/vg_srvnode-1_md1/lv_raw_md1' and 
/dev/vg_srvnode-1_md1/lv_main_swap1

Since we are getting this list from lvm commands, we assume that they are valid.
We have not validated that if these found cortx lvm paths are from our md disks or from some other.